vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -154,33 +148,63 @@ ent prj install --conflict-strategy=OVERRIDE
 ```
 
 ### Run a Project Locally
-1. Initialize Keycloak, which leverages docker-compose
+1. Initialize Keycloak, which leverages Docker Compose
 ``` sh
 ent prj ext-keycloak start
 ```
 2. Initialize backend microservices
 ``` sh
 ent prj be-test-run
 ```
+To fetch logs from bundle plugins, use the command
+```
+ent prj be-log
+```
+
 3. Initialize one or more frontend widgets, each from its own shell
 ``` sh
 ent prj fe-test-run
 ```
-See [this tutorial](../../tutorials/create/ms/run-local.md) for more details.
+Check out [Run Blueprint-generated Microservices and Micro Frontends in Dev Mode](../../tutorials/create/ms/run-local.md) for more details.
+
+### Bundle and Custom Resource Management
+Entando supports a series of `ent ecr` commands to manage bundles and custom resources. The following commands are particularly useful.
+
+Display the list of bundles associated with the current profile
+```
+ent ecr list
+```
+
+Generate a custom resource and deploy it to the current profile
+```
+ent ecr deploy
+``` 
+
+The helpers `get-bundle-id` and `get-plugin-id` have been added to `ent ecr` to calculate and display unique identifiers related to custom resources. This provides additional security controls around bundle-specific and microservice plugin resources.
```

Comment:
around bundle-specific and microservice plugin resources. >> slightly awkward, because bundle-specific is left hanging. sounds better if you switch, I think >> around microservice plugin and bundle-specific resources

Reply:
if plugin is singular i think it makes sense how it's written now, because switching it looks like we forgot the "s" and "plugin" should be plural. @nshaw can we change "microservice plugin resources" to "microservice plugins" here so this can read "This provides additional security controls around microservice plugins and bundle-specific resources." ??

Reply:
Works for me.